pygame_life.py

```diff
@@ -15,22 +106,20 @@
 
 
 def get_neighbours(grid: Grid, x: int, y: int) -> Neighbours:
-    """

```

Comment:
Please add back the documentation and update where you've changed the functions

Reply:
ok ok  thanku so much for guid